### PR TITLE
fix(package.json): fix command `rest-client-*` not found error

### PR DESCRIPTION
**Pull request by stegano**
Added list of missing commands to activationEvents entry

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,14 @@
     "onCommand:rest-client.copy-response-body",
     "onCommand:rest-client.switch-environment",
     "onCommand:rest-client.clear-aad-token-cache",
-    "onLanguage:http"
+    "onLanguage:http",
```
Comment (Huachao) [inline package.json]: Minor: move the `onLanguage:http` to the item of the list

Comment (stegano) [reply package.json]: Do you mean move `onLanguage:http` to tail of list?

Comment (Huachao) [reply package.json]: Yes 😄

**Comment by Huachao** [comment]
@stegano Merged, thanks

**Comment by Huachao** [comment]
@stegano I have published this in the latest release 0.23.1

**Pull request merged**